modules/core/client/app/init.js

```diff
@@ -1,80 +1,43 @@
-'use strict';
+(function (app) {
```

Comment:
>Can't the app.applicationModule name just be window.applicationConfiguration.applicationModuleName and same for dependencies?

It can, but I made it shorter.